setup.py

```diff
@@ -82,10 +83,10 @@ def read(*filenames, **kwargs):
         "Intended Audience :: Science/Research",
         "License :: Free for non-commercial use",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3.4",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
```

Comment:
Note there is a `Programming Language :: Python :: 3 :: Only` classifier. Should we include it? https://pypi.org/pypi?%3Aaction=list_classifiers

Reply:
Went ahead and added this as a fixup to the first commit. Note that `"3 " < "3."` lexicographically and so it goes at the top.